src/component/feature/FeatureToggleList/FeatureToggleListTable/FeatureToggleListTable.tsx

```diff
@@ -0,0 +1,196 @@
+    },
+    {
+        Header: 'Project ID',
+        accessor: 'project',
+        Cell: ({ value }: { value: string }) => (
+            <LinkCell to={`/projects/${value}`}>{value}</LinkCell>
+        ),
+        sortType: 'alphanumeric',
+    },
+    {
+        Header: 'State',
+        accessor: 'stale',
+        Cell: FeatureStaleCell,
+        sortType: 'boolean',
+    },
+    // Always hidden -- for search
+    {
+        accessor: 'description',
+    },
+];
+
+export const FeatureToggleListTable: VFC<IExperimentProps> = ({
+    data: input,
+    isLoading = false,
+}) => {
+    const data = useMemo(() => input, [input]);
+    const theme = useTheme();
+    const isSmallScreen = useMediaQuery(theme.breakpoints.down('md'));
+    const isMediumScreen = useMediaQuery(theme.breakpoints.down('lg'));
+    const ref = useLoading(isLoading);
+
+    const initialState = useMemo(
+        () => ({
+            sortBy: [{ id: 'createdAt', desc: false }],
+            hiddenColumns: ['description'],
+        }),
+        []
+    );
+
+    const {
+        getTableProps,
+        getTableBodyProps,
+        headerGroups,
+        rows,
+        prepareRow,
+        state: { globalFilter },
+        setGlobalFilter,
+        setHiddenColumns,
+    } = useTable(
+        {
+            columns,
+            data,
+            initialState,
+            sortTypes,
+            autoResetGlobalFilter: false,
+        },
+        useGlobalFilter,
+        useSortBy
+    );
+
+    useEffect(() => {
+        if (isSmallScreen) {
+            setHiddenColumns(['lastSeenAt', 'type', 'stale', 'description']);
+        } else if (isMediumScreen) {
+            setHiddenColumns(['lastSeenAt', 'stale', 'description']);
+        } else {
+            setHiddenColumns(['description']);
+        }
+    }, [setHiddenColumns, isSmallScreen, isMediumScreen]);
+
+    return (
+        <TablePanel
+            ref={ref}
+            header={
+                <TableToolbar title={`Feature Flags (${input.length})`}>
+                    <TableActions isSeparated onSearch={setGlobalFilter}>
+                        <Link
+                            component={RouterLink}
+                            to="/archive"
+                            underline="always"
+                            sx={{ marginRight: 3 }}
+                        >
+                            View archive
+                        </Link>
+                        <CreateFeatureButton
+                            loading={false}
+                            filter={{ query: '', project: 'default' }}
+                        />
+                    </TableActions>
+                </TableToolbar>
+            }
+        >
+            <SearchHighlightProvider value={globalFilter}>
+                <Table {...getTableProps()}>
+                    <SortableTableHeader headerGroups={headerGroups} />
+                    <TableBody {...getTableBodyProps()}>
+                        {rows.map(row => {
```

Comment:
There's no longer a visual indicator on the rows on mouseover. Do we still want that? @NicolaeUnleash ?